views/default/elgg.css.php

```diff
@@ -51,6 +52,7 @@
 echo elgg_view('elements/layout.css', $vars);
 echo elgg_view('elements/misc.css', $vars);
 echo elgg_view('elements/misc/spinner.css', $vars);
+echo elgg_view('elements/fa.css', $vars);
```

Comment:
I decided to place the hacks in this file at least for the time of testing/discussing. We can later place them somewhere else if necessary.